pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "Submit4DN"
-version = "2.2.0"
+version = "2.3.0"
```

Comment:
I would bump the version to 3.0.0 because this library cannot be used in Python 3. 6 and that's an incompatible change in the library, even if it won't affect the known uses.